### PR TITLE
Webhooks

### PR DESCRIPTION
**Pull request by benib**
Relations:
  - Issue: https://github.com/livingdocsIO/livingdocs-planning/issues/3543
  - Server: https://github.com/livingdocsIO/livingdocs-server/pull/2907
  - Editor: https://github.com/livingdocsIO/livingdocs-editor/pull/3445

### CODE REVIEW

File: reference-docs/server-configuration/webhooks.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+- `handle` needs to be unique on your project
+- `label` is used for UI only
+- `description` is only visible in this form and can be used to document the reason for this webhook or other humans to talk to for questions about the endpoint
+- `url` will be called with a POST request when a selected event happens
+- `token` is used to sign the request. Use this to [secure your webhooks](#securing-your-webhooks)
+- `active` needs to be `true` for this webhook to be called
+- `events` let's you define which events should trigger your webhook
+
+For testing purposes the service at https://webhook.site may come in handy. It gives you an URL you can use to send webhooks to and look at all the requests in a webinterface.
```
Comment (DaRaFF) [inline reference-docs/server-configuration/webhooks.md:25]: webhook.site is a cool service!

File: reference-docs/server-configuration/channel-config.md [hunk 2]
```diff
@@ -115,6 +115,22 @@ The following sample configuration file illustrates all of the above.
...
+
+    // Webhooks (added in release-2020-05)
+    webhooks: {
+      enabled: true
+      configurations: [{
+        handle: 'my-webhook',
+        label: 'My Webhook',
+        description: 'A description for future self and coworkers',
+        url: 'https://example.com/my-webhook-endpoint',
+        token: 'a-secret-token-to-sign-the-request'
```
Comment (DaRaFF) [inline reference-docs/server-configuration/channel-config.md:127]: I would point out it's a sha1 signed token.

File: reference-docs/server-configuration/webhooks.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+![Webhooks Configuration](./webhooks.png)
+
+Add a new Webhook by clicking "Add Webhook" or edit an existing one by clicking on it's entry in the list. This shows a form to configure the webhook.
+
+- `handle` needs to be unique on your project
+- `label` is used for UI only
+- `description` is only visible in this form and can be used to document the reason for this webhook or other humans to talk to for questions about the endpoint
+- `url` will be called with a POST request when a selected event happens
+- `token` is used to sign the request. Use this to [secure your webhooks](#securing-your-webhooks)
```
Comment (DaRaFF) [inline reference-docs/server-configuration/webhooks.md:21]: I would point out it's a sha1 signed token.

File: reference-docs/server-configuration/webhooks.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+```
+webhooks: {
+  enabled: true
+  configurations: [{
+    handle: 'my-webhook',
+    label: 'My Webhook',
+    description: 'A description for future self and coworkers',
+    url: 'https://example.com/my-webhook-endpoint',
+    token: 'a-secret-token-to-sign-the-request'
```
Comment (DaRaFF) [inline reference-docs/server-configuration/webhooks.md:41]: I would point out it's a sha1 signed token.

File: reference-docs/server-configuration/webhooks.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    "contentType": "regular",
+    "documentType": "article",
+    "eventType": "unpublish",
+    "publicationId": 112
+  }
+}
+```
+
+## Securing your webhooks
+If you have defined a `token` for your webhook, Livingdocs uses this to create a signature of the payload and sends it with the request in the HTTP header `x-livingdocs-signature`.
```
Comment (DaRaFF) [inline reference-docs/server-configuration/webhooks.md:94]: I would point out it's a sha1 signed token.

File: reference-docs/server-configuration/webhooks.md (hunk 1 above)
Comment (benib) [reply reference-docs/server-configuration/webhooks.md:25]: it is!

**Pull request merged**